### ISSUE TITLE
14.2 does not warn about dangers

### ISSUE THREAD

**Issue by JamMasterz** [hamsternz/IntroToSpartanFPGABook#15]
Shouldn't this chapter mention that when you leave the program running for a while, it will fry the segments? Because that's what just happened to me, 1 segment got fried, the other seven will probably follow in the near future.

**Comment by JamMasterz** [comment]
I assumed it got fried because it started blinking and then stopped working. But its actually a faulty display. After pushing and wiggling the segment with my nail, it started working again. Nevertheless, the segments get VERY hot when they are running for a while on program from chapter 14.2, so maybe a warning wouldn't hurt?

**Comment by hamsternz** [comment]
Agreed. Will add it on the weekend. You can drive them safely at low duty cycles (25%) or so. It is a design fault it the megawing,....  The basys2 is fine.
